tests/test_enterprise/views/test_course_enrollment_view.py

```diff
@@ -266,7 +265,114 @@ def test_get_course_enrollment_page_consent_declined(
             )
 
     @mock.patch('enterprise.views.render', side_effect=fake_render)
-    @mock.patch('enterprise.views.ProgramDataExtender')
+    @mock.patch('enterprise.views.CourseCatalogApiServiceClient')
+    @mock.patch('enterprise.views.EnrollmentApiClient')
+    @mock.patch('enterprise.api_client.ecommerce.ecommerce_api_client')
+    @mock.patch('enterprise.utils.Registry')
+    @ddt.data(True, False)
+    def test_get_course_enrollment_page_course_unenrollable(
+            self,
+            enrollable,
+            registry_mock,
+            ecommerce_api_client_mock,
+            enrollment_api_client_mock,
+            course_catalog_client_mock,
+            *args
+    ):  # pylint: disable=unused-argument
+        """
+        The message indicating that the course is currently unopen to new learners is rendered.
```

Comment:
@UmanShahzad "unopen" ---> "not open"??? :D

Reply:
Haha, well, turns out this exists in the Oxford Dictionary -- https://en.oxforddictionaries.com/definition/unopen

Reply:
That's crazy! :D